dev/ci/docker_linux/Dockerfile

```diff
@@ -96,10 +96,6 @@ RUN dpkg-query -L nodejs
 # Install Firebase
 # This is why we need nodejs installed.
 RUN /usr/bin/npm --verbose install -g firebase-tools
-# TODO(dnfield): Remove this once Firebase has a fix upstream for
```

Comment:
I'm not sure if we should remove this (the linked issue was closed because uploads were stable WITH this patch), but if we should, we should delete the `patch_firebase.sh` script too. @dnfield

Reply:
Right now I'm watching to see if https://cirrus-ci.com/task/5469977700466688 passes.

Reply:
Firebase was going to patch this in, this was a manual hack until they did.

Reply:
We've also mitigated this by just not uploading as much.

Reply:
Let's remove this and the patch.  If docs start timing out we can add it back, but I'm pretty sure we're covered now.  The underlying change is that we used to embed a git hash in every file in the docs, and now it's just in one file that's dynamically loaded by all the others.

Reply:
(maybe in a separate change, in case it does cause problems it's more easily reverted)